django_short_urls/models.py

```diff
@@ -161,8 +161,10 @@ def build_relative_path(self):
         return '/%s' % self.hash
 
     def build_absolute_uri(self, request):
-        """Builds the absolute url for the target link (including full server url)"""
-        return request.build_absolute_uri(self.build_relative_path())
+        """Builds the absolute url HTTPs for the target link (including full server url)"""
```

Comment:
It's already better than "use the scheme of the incoming API request", though ideally we could have an option for "force HTTPS", or "use the same scheme as the long_url shortened"?